packages/utils/src/isBlock.ts

```diff
@@ -51,6 +51,15 @@ const computedStyles = new WeakMap<Node, CSSStyleDeclaration>();
 export function isBlock(node: Node): boolean {
     let result: boolean;
     if (node instanceof Element) {
+        const tagName = node.tagName.toUpperCase();
+        // every custom jw-* node will be considered as blocks
+        if (tagName.startsWith('JW-') || tagName === 'T') {
+            return true;
+        }
+        // every custom td, tr node will be considered as blocks
+        if (tagName === 'TD' || tagName === 'TR') {
+            return true;
+        }
```

Comment:
@Zinston Christophe tells me that formatting spaces are not correctly removed around TD and TR without this. Since you worked a lot on `isBlock` and its related issues, could you please enlighten me/us ? Thanks !

Reply:
I adapted `blockTagNames` to ensure that so normally it should work without this extra code... Maybe he just didn't try since tables have been merged?

Reply:
I doubt this PR has been rebased on master since tables were merged, yes.